packages/documentation/copy/en/developer/Getting_Started_Linux.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Getting started on Linux 
+layout: docs
+permalink: /docs/handbook/download
+oneline: "Quick start with Lingua Franca on Linux"
+preamble: >
+---
+
+# Getting started on Linux
+
+## Installation
+
+### Debian & Other
+
+Sadly there are currently no packaged version of the lingua-franca compiler in debian distributions so there is
+really only the manual way. But we offer prebuild binaries [here](https://github.com/lf-lang/lingua-franca/releases). 
+Please note that you need a modern version of java 11 or greater.
```

Comment:
```suggestion
Please note that you need a modern version of java 17 or greater.
```

Also, it seems you contradict yourself. You say there are no packaged versions and yet there a pre-built binaries... ???

Reply:
A pre-built binary is not the same thing as a package. A package can be installed using a package manager. A pre-built binary can be executed without having to compile it first.